src/Run/Setup.cs

```diff
@@ -77,40 +121,83 @@ private string FindSettingType(string valueToFind)
             return null;
         }
 
-        public void prepareValues(string os, Dictionary<string, string> parameters, string configFile)
+        public void PrepareValues(string os, Dictionary<string, string> parameters, string configFile)
         {
             SettingParameters = new Dictionary<string, string>(parameters);
             Os = os;
             ConfigurationFilePath = configFile;
+            // Add defaults for run tool settings if they haven't been define din config.json
+            SetRunToolSettingsDefaults();
+            ToolSettings = Settings.Where(s => s.Value.ValueType == null ||
+                                               s.Value.ValueType.Equals("runToolSetting")
+                                               ).ToDictionary(s => s.Key, s => string.Empty) ?? new Dictionary<string, string>();
+            // A dev may have overriden the default values for a tool setting, but not specified the ValueType
+            foreach(var key in ToolSettings.Keys)
+            {
+                Settings[key].ValueType = "runToolSetting";
+            }
+        }
+
+        private void SetRunToolSettingsDefaults()
+        {
+            // If RunQuiet is already defined in config.json, don't override it
+            if (!Settings.ContainsKey(runQuietReservedKeyword))
+            {
+                Setting runQuietSetting = new Setting()
+                {
+                    Values = new List<string>() { "True", "False" },
+                    ValueType = "runToolSetting",
+                    Description = "Run tool specific setting.  Set to True to only display output from the executing command.  Default, False",
+                    DefaultValue = "false"
+                };
+
+                Settings.Add(runQuietReservedKeyword, runQuietSetting);
+            }
+
         }
 
         public int ExecuteCommand(string commandSelectedByUser, List<string> parametersSelectedByUser)
         {
+            ParseRunToolSettings(commandSelectedByUser);
+            string runQuietValue;
+            bool runQuiet = false;
+            if (ToolSettings.TryGetValue(runQuietReservedKeyword, out runQuietValue))
+            {
+                runQuiet = runQuietValue.Equals("true", StringComparison.OrdinalIgnoreCase) ? true : false;
```

Comment:
the `? true : false` isn't actually needed here as the value of .Equals will be exactly what you want :)